packages/api/src/event/service.ts

```diff
@@ -10,10 +11,14 @@ import {
 
 export const createEvent = async (data: CreateEventRequest): Promise<void> => {
   if (data.body.analytics) {
+    const { browser, os, device } = UAParser(data.userAgent);
     try {
       const analyticsData: { [key: string]: unknown; distinct_id?: string } =
         data.body.analytics.data;
       analyticsData.distinct_id = data.body.analytics.distinctId;
+      analyticsData["$browser"] = browser.name;
+      analyticsData["$os"] = os.name;
+      analyticsData["$device"] = device.type;
```

Comment:
Does Mixpanel require these names?

Reply:
Yes, I think because its UI surfaces these fields separately from the event body in general